nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -12,6 +12,33 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.1.0"]
+== Release notes for {secondary-scheduler-operator-full} 1.1.0
+
+Issued: 2022-9-1
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.1.0:
+
+* link:https://access.redhat.com/errata/RHSA-2022:6152[RHSA-2022:6152]
```

Comment:
I get a 404 error when I try to access this URL. Could you please check and rectify it?

Reply:
@sayjadha Apologies, I should have stated this when I requested peer review - this URL won't work until the day the advisory ships. So I will check that it works before actually merging this PR on 9/1. Thanks!

Reply:
That makes sense. Thank you for the clarification!